src/bigquery.ts

```diff
@@ -36,6 +36,25 @@ async function queryWithRetries(query: string, retriesDone = 0): Promise<void> {
     }
 }
 
+export function buildInsertQuery(
+    table: string,
+    data: Record<string, string | number | boolean | undefined>,
+): string {
+    const keys = Object.keys(data).join(', ').trim()
+    const values = Object.values(data)
+        .map((value) => JSON.stringify(value))
```

Comment:
Hvis value er undefined, burde vi vel sette `null` i BigQuery-tabellen. Skjer det nå, eller må vi gjøre noe for å tvinge det? Har sett at `undefined` har dukket opp i tabellen tidligere når data mangler (f.eks platform).

Reply:
Tom verdi skal fungere som null, om eg ikkje tar heilt feil. Grunnen til at me får `'undefined'` i dag er fordi den tidlegare koden kun bruk template strings, og \`${undefined}\` = `'undefined'`. Men `JSON.stringify(undefined)` blir kun `undefined`.